pkg/virt-controller/watch/vm.go

```diff
@@ -3027,3 +3037,12 @@ func (c *VMController) setupLiveFeatures(
 	c.setupCPUHotplug(vm, vmi, VMIDefaults, maxRatio)
 	c.setupMemoryHotplug(vm, vmi, maxRatio)
 }
+
+func (c *VMController) patchVMITerminationGracePeriod(gracePeriod *int64, vmi *virtv1.VirtualMachineInstance) error {
+	if gracePeriod == nil {
+		return fmt.Errorf("VM %s has no DeletionGracePeriod, there is nothing to patch", vmi.Name)
```

Comment:
I disagree with this! If there is nothing to patch we should skip the patch and go forward without any problems. WDYT? Thanks

Reply:
That makes sense to me, I wasn't completely sure whether to return an error here or continue but it makes sense to continue to stop the VMI using its existing grace period.